httpmw/cognito_user.go

```diff
@@ -0,0 +1,34 @@
+package httpmw
+
+// CognitoUser cogntito user entity.
+type CognitoUser struct {
+	Username string
+	Groups   map[string]struct{}
```

Comment:
Since we're using a Get/Set pattern, I think we should make these fields private. Wdyt?

```suggestion
	username string
	groups   map[string]struct{}
```

Reply:
I was thinking about that as well. Makes sense to me.

Reply:
Updated.